src/System.Memory/tests/ReadOnlySpan/CtorArray.cs

```diff
@@ -89,16 +89,6 @@ public static void CtorArrayNullArray()
         }
 
         [Fact]
-        public static void CtorArrayWrongArrayType()
```

Comment:
It would be better to keep this test, but make it test the right condition - that it is possible to create the Span from variant arrays.